cost-analyzer/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "1.108.0"
+appVersion: "development"
```

Comment:
What I like about this change is that it defaults to the nightly images when you are installing the chart from your filesystem, which will help when we are doing local chart dev. In contrast, releases are always built via CI so it is possible to have logic there that alters this and feel confident about the coverage there

Reply:
Yes, that is my suggestion for modifying the automated release process.